### PR TITLE
Dockerization rework

### PR DESCRIPTION
**Pull request by hbollon**
Based on @danduk82 work
With this PR the Docker Compose environment is fully functional
I also improved the `fixturize` command which now do `update_or_create` operations in order to be compatible with already populated databases (it caused some issues in case of restart of the migrate container in the docker-compose)

### CODE REVIEW

**Review by lanseg — APPROVED** [review]

**Review by danduk82 — APPROVED** [review]
LGTM

**Comment by maltaesousa** [comment]
This is nice, thanks. The migration container is a good idea. I've tested with our composition and it works great

File: Dockerfile [hunk 1]
```diff
@@ -15,11 +17,7 @@ ENV C_INCLUDE_PATH=/usr/include/gdal
...
+RUN mv /app/geoshop_back/default_settings.py /app/geoshop_back/settings.py
 
-ARG ENV_FILE
-RUN mv ${ENV_FILE} .env && mkdir /mnt/geoshop_data
-
-RUN export $(egrep -v '^#' .env | xargs) && \
-    python manage.py migrate && \
-    python manage.py collectstatic --noinput && \
-    python manage.py compilemessages --locale=fr
+# FIXME: why was this needed?
```
Comment (maltaesousa) [inline Dockerfile:22]: You can delete this, it was a first try to put some static files as a mount. For instance, images of metadata that are also used in other apps. But we decided to serve those files from another server.

Comment (hbollon) [reply Dockerfile:22]: Alright I remove that :ok_hand:

**Pull request merged**